3주차/FindAnimal/src/pages/Page3.jsx

```diff
@@ -0,0 +1,52 @@
+import { useState } from "react";
+import {
+  AnswerButton,
+  TextBox,
+  Group,
+  QuestionBox,
+  NavigateButton,
+} from "../style/commonStyle";
+
+function Page3({ goBack, goForward }) {
+  const [selectedOption, setSelectedOption] = useState(null);
+  const [buttonActive, setButtonActive] = useState(false);
+
+  const handleOptionChange = (option) => {
+    setSelectedOption(option);
+    setButtonActive(option != null);
+  };
+
+  const handleGoForward = () => {
+    goForward(selectedOption);
+  };
+
+  return (
+    <div>
+      <Group>
+        <TextBox>귀엽다는 말 들어본 적 있어?</TextBox>
+        <QuestionBox>
+          <AnswerButton
+            answer="응!!"
+            onClick={() => handleOptionChange("웅!")}
+          />
+          <AnswerButton
+            answer="아니."
+            onClick={() => handleOptionChange("아니ㅠ")}
+          />
+        </QuestionBox>
```

Comment:
각 페이지에 보면 QuestionBox랑 AnswerButton처럼 반복되는 구조들이 있는데 요런 애들을 컴포넌트로 만들어서 재사용하면 너무 좋은 구조가 되겠다!

Reply:
예를 들면 QuestionBox라는 컴포넌트를 components 디렉토리 아래에 만들어서
```suggestion
        <QuestionBox 
             answer={["웅!", "아니ㅠ"]} 
             handlOptionChange={handlOptionChange}
             />       
```
이렇게 넘겨주고,
```
const QuestionBox = ({answer, handleOptionChange}) => {
// 여기서 answer을 map 돌려서 AnswerButton을 만드는 방식
}
```
이런식으로!